coreos-base/coreos-metadata/files/coreos-metadata.target

```diff
@@ -0,0 +1,3 @@
+[Unit]
```

Comment:
Is this a bad idea? My original thinking was to create a general target that people would use to install services under and depend on. If it has coreos-metadata.service by default, it means that people will have to create a drop-in to modify it or another target altogether. Maybe we should just leave this out.